hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/CryptoQueryHandler.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.service.token;
+
+import com.hedera.node.app.spi.QueryHandler;
+import com.hedera.services.store.models.Account;
+import com.hederahashgraph.api.proto.java.*;
+import java.util.Optional;
+import org.checkerframework.checker.nullness.qual.NonNull;
```

Comment:
Please use jetbrains annotations (see https://github.com/hashgraph/hedera-services/issues/4234)

Reply:
Since this is an open source project, I'd prefer JSR 305, too.

Reply:
jsr305 is very problematic when using java modules since the `javax.annotation` package is used by several modules. See https://nipafx.dev/jsr-305-java-9/

Reply:
Isn't it only problematic, when you use JavaEE?